src/incident_reports/incident_reports_server/controllers/incident_reports_controller.py

```diff
@@ -0,0 +1,135 @@
+from flask import Flask, jsonify, request, make_response
+import argparse
+import os
+
+from incident_reports_server.application.services import Services
+from incident_reports_server.factory.incident_report_factory import IncidentReportFactory
+from incident_reports_server.models.models import IncidentReportResponse, ResponseMessages
+from incident_reports_server.validators.incident_report_validator import IncidentReportValidator
+
+def create_app(persistence=None):
+    app = Flask(__name__)
+    
```

Comment:
Enable CORS so React and Angular can access the backend:
- Import cors
```py
from flask_cors import CORS
```
- Enable cors
```py
CORS(app)
```
_We let all origins can access incident_reports service for now_